script.js

```diff
@@ -1,4 +1,137 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// Use Math to generate system's generated input for the game
+const getRandomHand = () => {
+  const systemGenerateNumber = 1 + Math.floor(Math.random() * 3);
+  if (systemGenerateNumber === 1) {
+    return "scissors";
+  } else if (systemGenerateNumber === 2) {
+    return "paper";
+  } else return "rock";
+};
```

Comment:
Like the way you are naming your variables. 
Although Basics only introduce the `var` keyword, do keep up your exploration of learning new things! 

On the note of naming, although it is clear what you are trying to convey, to shorten to something like `sysGenerateNum` would help to minimize length of code, which always helps where necessary